vision/unit_tests/test_coordinate_lengths.py

```diff
@@ -0,0 +1,151 @@
+        function_return_at_zero_degrees_latitude = latitude_length(basic_input_in_radians)
+        # Tests to insure the expected output is the same as the actual output, given the same input
+        self.assertEqual(
+            latitude_length_at_zero_degrees_latitude, function_return_at_zero_degrees_latitude
+        )
+
+    # Testing longitude_length
+
+    # Tests the maximum possible latitude in degrees as an input to the latitude_length function
+    def test_longitude_length_at_max_val(self) -> None:
+        """
+        Asserts that longitude_length returns expected value at maximum input (90 degrees)
+
+        Returns
+        -------
+        None
+        """
+        max_input_in_degrees = 90
+        max_input_in_radians: float = np.deg2rad(max_input_in_degrees)
+        longitude_length_at_max_latitude: float = (
+            111412.84 * np.cos(max_input_in_radians)
+            - 93.5 * np.cos(3 * max_input_in_radians)
+            + 0.118 * np.cos(5 * max_input_in_radians)
+        )
+        function_return_at_max_latitude = longitude_length(max_input_in_degrees)
+        # Tests to insure the expected output is the same as the actual output, given the same input
+        self.assertEqual(longitude_length_at_max_latitude, function_return_at_max_latitude)
+
+    # Tests the minimum possible latitude in degrees as an input to the latitude_length function
+    def test_longitude_length_min_val(self) -> None:
+        """
+        Asserts that longitude_length returns expected value at minimum input (-90 degrees)
+
+        Returns
+        -------
+        None
+        """
+        max_input_in_degrees = -90
+        min_input_in_radians: float = np.deg2rad(max_input_in_degrees)
+        longitude_length_at_min_latitude: float = (
+            111412.84 * np.cos(min_input_in_radians)
+            - 93.5 * np.cos(3 * min_input_in_radians)
+            + 0.118 * np.cos(5 * min_input_in_radians)
+        )
+        function_return_at_min_latitude = longitude_length(max_input_in_degrees)
+        # Tests to insure the expected output is the same as the actual output, given the same input
+        self.assertEqual(longitude_length_at_min_latitude, function_return_at_min_latitude)
+
+    # Tests a basic input in degrees as an input to the latitude_length function
+    def test_longitude_length_at_zero(self) -> None:
+        """
+        Asserts that longitude_length returns expected value at a basic input (0 degrees)
+
+        Returns
+        -------
+        None
+        """
+        basic_input_in_degrees = 0
+        basic_input_in_radians: float = np.deg2rad(basic_input_in_degrees)
+        longitude_length_at_zero_degrees_latitude: float = (
+            111412.84 * np.cos(basic_input_in_radians)
+            - 93.5 * np.cos(3 * basic_input_in_radians)
+            + 0.118 * np.cos(5 * basic_input_in_radians)
+        )
+        function_return_at_zero_degrees_latitude = longitude_length(basic_input_in_radians)
```

Comment:
Type hints for `basic_input_in_degrees` and `function_return_at_zero_degrees_latitude`